ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly-4.18-cpou-upgrade-from-4.15.yaml

```diff
@@ -32,13 +32,13 @@ base_images:
     namespace: ci
     tag: latest
 releases:
-  intermediate416:
```

Comment:
If I remember right, we keep the intermediate release name with version number on purpose, to help us understanding the upgrade path, also align with other similar upgrade ci jobs.  cc @jiajliu

Reply:
yes, this style is inherited from chain upgrade.

Reply:
Those names are just pointers to the release payloads. Any name should work as long the reference use the same name.
Actually I have the plan the change the chain upgrade after this one. ( The original plan is change chain upgrade together with this one, change the mind later as I want to make changes small, simple and specific ).